packages/webapp/src/components/Form/FormNavigationButtons/index.tsx

```diff
@@ -70,14 +74,15 @@ const FormNavigationButtons = ({
           sm
           fullLength
         >
-          {isFinalStep ? (
-            t('common:SAVE')
-          ) : (
-            <>
-              <span>{t('common:NEXT')}</span>
-              <ChevronRight />
-            </>
-          )}
+          {continueButtonContent ||
```

Comment:
In this case it's irrelevant because there's only one step, but for complete flexibility and to avoid confusion should we allow customization of both the "Next" button and the "Save" button, with separate "content" props?